spec/order_takeaway_spec.rb

```diff
@@ -0,0 +1,19 @@
+require './lib/order'
+require './lib/takeaway'
+
+describe Order do
+  let(:test_order) { Order.new }
+
+  describe '#add_item' do
+    it 'adds item to basket' do
+      expect(test_order.add_item("quesidilla",1)).to eq 1
```

Comment:
Could you create a second test to check that the basket also contains "quesidilla" as well as quantity 1.